superset/dataframe.py

```diff
@@ -30,6 +48,47 @@ def size(self):
     def data(self):
         return self.__df.to_dict(orient='records')
 
+    @classmethod
+    def db_type(cls, dtype):
```

Comment:
yes to be explicit I should differentiate between col_numpy_type and col_db_type, which I'm converting here...